### ISSUE TITLE
"{filename}" does not work properly when inserting an image.

### ISSUE THREAD

**Issue by ghost** [getpelican/pelican#1497]
.md file
```
![images]({filename}/images/dxp.png)
```
output:
```
<img alt="images" src="{filename}/images/dxp.png" style="width: 640px; height: auto; max-width: 100%;">
```
`make html` waning MSG：
>
WARNING: Unable to find ParseResult(scheme='', netloc='', path='/images/dxp.png'
, params='', query='', fragment=''), skipping url replacement

version info
```
$ pip show pelican
---
Name: pelican
Version: 3.4.0

$ python --version
Python 3.4.1
(blogsite)
```
os:win64bit pro
================================
solution:
I just remove {filename} and modify `better_figures_and_images` plugin's warning msg,that's ok.

**Comment by jonathan-s** [comment]
@justinmayer This issue could be closed, as it was implemented in 3.5.0 according to the reference.

**Comment by justinmayer** [comment]
Thanks, Jonathan.
